frontend/src/components/Accounts/AccountDetailsContent.vue

```diff
@@ -153,7 +153,7 @@
 					/>
 				</template>
 			</Accordion>
-			<Accordion v-if="account.tokens">
+			<Accordion v-if="account.tokens && account.tokens.length">
```

Comment:
In the other parts of the code it seems that a more explicit check is being done `.length > 0`. 

While it is semantically equivalent for this, the latter is more explicit.